napari/layers/points/points.py

```diff
@@ -1208,46 +1212,38 @@ def selected_data(self, selected_data):
             self._set_highlight()
             return
         index = list(self._selected_data)
-        edge_colors = np.unique(self.edge_color[index], axis=0)
-        if len(edge_colors) == 1:
-            edge_color = edge_colors[0]
+        unique_edge_color = _unique_element(self.edge_color[index])
+        if unique_edge_color is not None:
```

Comment:
This (and the other similar places) feels like a good place for the walrus operator.

```suggestion
        if unique_edge_color := _unique_element(self.edge_color[index]):
```

What do you think?

Reply:
Oh wow, I actually forgot we ditched python 3.7! Awesome, I'm gonna stick walruses everywhere now :D